.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -634,6 +661,8 @@ stages:
 
       - script: tracer\build.cmd BuildAndRunNativeUnitTests
         displayName: Build and Test
+        env:
+          DD_LOGGER_DD_API_KEY: $(ddApiKey)
```

Comment:
```suggestion
```

Reply:
secrets cannot be set `globally` (doesn't map to env-vars) so we have to do it in each job.